source/jormungandr/jormungandr/street_network/valhalla.py

```diff
@@ -140,36 +112,40 @@ def _get_response(self, json_resp, mode, pt_object_origin, pt_object_destination
         resp.status_code = 200
         resp.response_type = response_pb2.ITINERARY_FOUND
 
-        for leg in json_resp['trip']['legs']:
-            journey = resp.journeys.add()
-            journey.duration = leg['summary']['time']
-            if clockwise:
-                journey.departure_date_time = datetime
-                journey.arrival_date_time = datetime + journey.duration
-            else:
-                journey.departure_date_time = datetime - journey.duration
-                journey.arrival_date_time = datetime
+        journey = resp.journeys.add()
```

Comment:
We used to add one journey per leg (which is a section).
It was invisible as we are supposed to have only one leg, but it was not consistent. I did the cleanup along the comparison while I tried to use pretty much the same thing in geovelo.